extensions/gitlab/src/cache.ts

```diff
@@ -136,7 +136,7 @@ export function useCache<T>(
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [timestamp, setTimestamp] = useState<Date>(new Date());
   const shouldForceRefetch = useRef(false);
-  const depsAll = [timestamp];
+  const depsAll: any[] = [timestamp];
```

Comment:
Why you changed it to `any[]`?

Reply:
Without the change, there is actually a compilation error in the current `main` branch:

![image](https://user-images.githubusercontent.com/3402603/180654399-2c1462bd-d112-40a7-b480-ce53aa451269.png)

This is because the items that are pushed are contianed in the type `type DependencyList = ReadonlyArray<unknown>;`. Upon further inspection, it seems better to type this as `unknown[]`, so I've changed it 👀